### ISSUE TITLE
Building Android ARM64 locally

### ISSUE THREAD

**Issue by michaeltheory** [eclipsesource/J2V8#520]
Hey,

I was able to get a good ways through the build process. I believe it compiled V8 succesfully. However, I eventually reach an error 

`Error response from daemon: No such container: j2v8.android.arm64`

Interestingly, I was able to build arm64 in version 7.4 but I can't remember if that was because I enabled it somehow or it always worked.

**Comment by michaeltheory** [comment]
Update: Even a regular arm build gives the same `Error response from daemon: No such container: j2v8.android.arm`

**Comment by ahmadov** [comment]
Hi @michaeltheory,

could you please give more information about the environment?
What command do you use to build J2V8?

**Comment by michaeltheory** [comment]
I'm using Window command line with Docker 

`C:\Python27\python.exe build.py -t android -a arm64 -dkr` As I mentioned above arm or arm64 both hit the same error

**Comment by ahmadov** [comment]
Hi @michaeltheory,
sorry for the late response. Do you still have the same issue? Do you use Windows container for docker?
I haven't tried to build on Windows but I would try to have time to reproduce on Windows if this issue still persists.

**Comment by michaeltheory** [comment]
@ahmadov yep -- same issue. I installed Docker on Windows, and I believe it uses WSL2 backend
